hack/env.sh

```diff
@@ -30,4 +30,8 @@ export GOOS
 export GOARCH
 
 export GO111MODULE=on
-export GOPROXY=https://goproxy.io
+
```

Comment:
I'm wondering that whether we need `GOPROXY` both are defined in the Makefile and env.sh.

Reply:
/cc @allencloud

Reply:
Any updates? @SataQiu